packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -352,19 +351,19 @@ function commitLifeCycles(
       return;
     }
     case SuspenseComponent: {
-      if ((finishedWork.mode & StrictMode) === NoEffect) {
-        // In loose mode, a placeholder times out by scheduling a synchronous
-        // update in the commit phase. Use `updateQueue` field to signal that
-        // the Timeout needs to switch to the placeholder. We don't need an
-        // entire queue. Any non-null value works.
+      if ((finishedWork.mode & StrictMode) === NoContext) {
+        // In loose mode, a suspense boundary times out by scheduling a
+        // synchronous update in the commit phase. Use `updateQueue` field to
+        // signal that the Timeout needs to switch to the placeholder. We don't
+        // need an entire queue. Any non-null value works.
         // $FlowFixMe - Intentionally using a value other than an UpdateQueue.
         finishedWork.updateQueue = emptyObject;
         scheduleWork(finishedWork, Sync);
       } else {
         // In strict mode, the Update effect is used to record the time at
-        // which the placeholder timed out.
-        const currentTime = requestCurrentTime();
-        finishedWork.stateNode = {timedOutAt: currentTime};
+        // which the children timed out.
+        // $FlowFixMe - Intentionally using a value other than an UpdateQueue.
+        finishedWork.updateQueue = {timedOutAt: requestCurrentTime()};
```

Comment:
If we use this field for something other than the UpdateQueue, we must never use this field in any generic branches. I.e. we can't have any code that checks `.updateQueue` for a bunch of different or all tag types.

What's your motivation though? This doesn't make sense to me. It is a signal that stores a stateful value over time and never is allowed to be reset when fully processed. Sounds like state to me. Either stateNode or memoizedState.